package.json

```diff
@@ -33,93 +33,76 @@
+    "@commitlint/cli": "^7.0.0",
+    "@commitlint/config-conventional": "^7.0.1",
+    "@types/node": "^10.5.4",
+    "cheerio": "^1.0.0-rc.2",
+    "commitizen": "^2.10.1",
+    "cross-env": "^5.2.0",
+    "cz-conventional-changelog": "^2.1.0",
+    "fuse-test-runner": "^1.0.16",
     "gulp": "3.9.1",
-    "gulp-bump": "2.4.0",
-    "gulp-clean": "0.3.2",
-    "gulp-concat": "2.6.0",
-    "gulp-header": "^1.8.8",
-    "gulp-rename": "1.2.2",
-    "gulp-replace": "0.5.4",
-    "gulp-sourcemaps": "2.6.0",
-    "gulp-typescript": "3.1.5",
-    "gulp-uglify": "2.0.0",
-    "gulp-wrap": "0.13.0",
+    "gulp-bump": "3.1.1",
+    "gulp-clean": "0.4.0",
+    "gulp-header": "^2.0.5",
+    "gulp-rename": "1.4.0",
+    "gulp-replace": "1.0.0",
+    "gulp-sourcemaps": "2.6.4",
+    "gulp-typescript": "5.0.0-alpha.3",
+    "gulp-uglify": "3.0.1",
+    "gulp-wrap": "0.14.0",
     "homedir": "^0.6.0",
-    "husky": "^0.14.3",
-    "jsdom": "^9.12.0",
-    "mkdirp": "^0.5.1",
-    "mocha": "^3.1.0",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
-    "reflect-metadata": "^0.1.8",
-    "run-sequence": "^1.2.2",
-    "should": "^11.1.0",
-    "sinon": "^1.17.6",
+    "jsdom": "^11.12.0",
+    "mocha": "^5.2.0",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "reflect-metadata": "^0.1.12",
+    "run-sequence": "^2.2.1",
+    "sinon": "^6.1.4",
     "typescript": "^3.1.0-dev.20180804",
-    "uglify-js": "^3.1.8",
-    "validate-commit-msg": "^2.14.0",
-    "wires-reactive": "^1.1.1"
+    "uglify-js": "^3.4.6",
+    "wires-reactive": "^1.1.32"
   },
   "dependencies": {
-    "acorn": "^5.1.2",
-    "acorn-jsx": "^4.0.1",
+    "acorn": "^5.7.1",
+    "acorn-jsx": "^4.1.1",
     "ansi": "^0.3.1",
-    "app-root-path": "^2.0.1",
-    "base64-img": "^1.0.3",
-    "base64-js": "^1.2.0",
-    "chokidar": "^1.6.1",
-    "clean-css": "^4.1.9",
-    "escodegen": "^1.8.1",
-    "express": "^4.14.0",
-    "fliplog": "^0.3.13",
-    "fs-extra": "^2.0.0",
+    "app-root-path": "^2.1.0",
+    "base64-img": "^1.0.4",
+    "base64-js": "^1.3.0",
+    "chokidar": "^2.0.4",
+    "clean-css": "^4.1.11",
+    "escodegen": "^1.11.0",
+    "express": "^4.16.3",
+    "fliplog": "^1.0.4",
+    "fs-extra": "^7.0.0",
     "fuse-concat-with-sourcemaps": "^1.0.5",
-    "getopts": "^2.1.1",
-    "glob": "^7.1.1",
-    "ieee754": "^1.1.8",
-    "inquirer": "^3.0.6",
-    "lego-api": "^1.0.7",
+    "glob": "^7.1.2",
+    "ieee754": "^1.1.12",
+    "inquirer": "^6.0.0",
+    "lego-api": "^1.0.8",
+    "minimist": "^1.2.0",
     "mustache": "^2.3.0",
-    "postcss": "^6.0.1",
-    "pretty-time": "^0.2.0",
-    "prettysize": "0.0.3",
+    "postcss": "^7.0.1",
+    "pretty-time": "^1.1.0",
+    "prettysize": "1.1.0",
     "realm-utils": "^1.0.9",
-    "regexpu-core": "^4.1.3",
-    "request": "^2.79.0",
+    "request": "^2.87.0",
     "shorthash": "0.0.2",
-    "source-map": "^0.7.1",
-    "tslib": "^1.8.0",
-    "watch": "^1.0.1",
-    "ws": "^1.1.1"
```

Comment:
That's a huge bump from 1 to 5
Did you verify the build yourself?

Reply:
Which one?